libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java

```diff
@@ -483,11 +477,43 @@ private void updateProgressChangeListener() {
     }
   }
 
+  /**
+   * Updates the size of the bounding box used to determine which route line was clicked. Upon
+   * a map click an attempt will be made to find a route line intersecting the bounding box. This
+   * is used to select alternative route lines on the map.
+   *
+   * Passing large integers values through this method means that a user doesn't
+   * have to be very accurate with tapping on an actual route line because the invisible
+   * box is large and thus, there's a good chance that a portion of a route line will run
+   * through the query box's area. A smaller rectangle will decrease the chances.
+   */
+  public void updateClickDistancePadding(int newRouteClickRectFPadding) {
+    routeClickPadding = newRouteClickRectFPadding;
+  }
```

Comment:
Same as above, does this need to be dynamic? Is there a use-case? If not, let's leave it immutable and provided by the builder only.

Reply:
See my comment below.